docs/network-drivers.md

```diff
@@ -213,6 +213,40 @@ The body of the api call request will look like:
 The brocade driver accepts interface names the same way they would be accepted
 in the console of the switch, ex. ``101/0/10``.
 
+### Dell Networking OS 9 driver
+
+#### switch_register
+
+The ``type`` field for the Dell Networking OS 9 driver has the value:
+
+    http://schema.massopencloud.org/haas/v0/switches/dellnos9
+
+In addition to ``type``, you need to supply ``username``, ``password``,
+``hostname``, and ``interface_type``. ``interface_type`` is of the form
+"GigabitEthernet", "TenGigabitEthernet". Please look at the switch guide for
+more valid interface types.
+
+The API server either runs on port 8008 (HTTP) or 8888 (HTTPS), so be sure to
```

Comment:
I would say the switch's API server to make this a bit more clear -- it took me a second to figure out you weren't talking about the HIL API server, and I was momentarily confused.